src/app/utils/accountPageUtils.js

```diff
@@ -2,6 +2,9 @@
 import axios from 'axios';
 
 import appConfig from '../data/appConfig';
+import { CLOSED_LOCATION_REGEX } from '../data/constants';
+
+export const isClosed = optionInnerText => !!optionInnerText.match(CLOSED_LOCATION_REGEX);
```

Comment:
Is there a reason to cast this as a boolean?

Reply:
Is there a different approach you were thinking? Merging for now. I wanted to write very simple unit tests for this.